### DISCUSSION TITLE
Cannot change status after force

### DISCUSSION THREAD

**Discussion by andrew-manger** [VueTorrent/VueTorrent#2046] [Q&A] [answered]
When I force download a torrent, I can't change the status anymore in the VueTorrent UI. For example, if I force download/seed and then try to resume it or pause it, the UI doesn't respond. The status won't change. I need to go back to the standard qBT UI to manipulate it.

 I am using qBT in a docker container and accessing it through the browser.

**Comment by Larsluph** [comment]
Can you check your DevTools to see if you have errors that may help us diagnose your issue?

Checkout [the wiki](https://github.com/VueTorrent/VueTorrent/wiki/Support) for help

**Comment by andrew-manger** [comment]
Thanks! Here is the log
[1732782482643.log](https://github.com/user-attachments/files/17945025/1732782482643.log)

**Reply by Larsluph** [reply, marked as answer]
Seems like you receive 404 on those API calls, are you using qBit 5.X?

We fetch the current version on login/startup, make sure you reload your page after each update.

**Reply by andrew-manger** [reply]
Sorry for the late reply! I changed the image I was using for qbit and everything is chill now. I'm assuming there must have been a mismatch of version. Thanks so much!
